.github/workflows/purefinance_deployment copy.yaml

```
@@ -0,0 +1,39 @@
+name: Deploy Purefinance
+
+on:
+  workflow_dispatch:
+    push:
+        branches:
+        - main
```

This is not properly indented and the default branch is `master`.

```suggestion
  push:
    branches:
    - master
```